src/components/IndexRoute.tsx

```diff
@@ -3,11 +3,13 @@ Copyright Juhani Vähä-Mäkilä (juhani@fmail.co.uk) 2022.
 Licenced under EUROPEAN UNION PUBLIC LICENCE v. 1.2.
  */
 import React from 'react'
+import { useTranslation } from 'react-i18next'
 
 export default function IndexRoute() {
+  const { t } = useTranslation()
```

Comment:
```suggestion
  const { translation } = useTranslation()
```

Reply:
`t` comes here from object destructuration. One can change this by either wrapping `useTranslation` hook or renaming destructured property (`{ t: translate } = useTranslation()`)

Reply:
Imo `{ t }` is still best :)